canonical-websites/maas.io
# Update to Vanilla v1.6.3

## Done
Update the Vanilla dep to v1.6.3. Hot fixed the issue with max-width.

## QA
- Pull this branch
- Run `./run`
- View the site locally in your web browser at: http://0.0.0.0:8006/
- Click through the pages and check it looks the same as live.

Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1464
